src/material-experimental/mdc-tabs/ink-bar.ts

```diff
@@ -128,16 +141,36 @@ export class MatInkBarFoundation {
     this._destroyed = true;
   }
 
+  /** Creates and appends the indicator element. */
```

Comment:
This should also explain what an "indicator" is.

Reply:
Since indicator and ink bar are synonymous, I renamed our use of the indicator to be consistently called the "ink bar". Alternatively we can rename everything to "indicator" (and also rename classes like `MatInkBar` to `MatTabIndicator`).

Where we do mention indicator (in comments referring to MDC's indicator), I made it clearer that we named it ink bar.